qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -27,6 +27,10 @@
 	UndefinedVariable("`{0}` cannot be resolved to a variable."), //
 	UnkwownType("`{0}` cannot be resolved to a type."), //
 	UnkwownMethod("`{0}` cannot be resolved or is not a method of `{1}` Java type."), //
+	InvalidMethodVoid("Invalid `{0}` method of `{1}` Java type : void return is not allowed."), //
+	InvalidMethodHasParameters("Invalid `{0}` method of `{1}` Java type : method with parameters is not allowed."), //
+	InvalidMethodFromObject("Invalid `{0}` method of `{1}` Java type : method from java.lang.Object is not allowed."), //

```

Comment:
Can we drop the "Java type" parts? Feels verbose

Reply:
fixed